app/decorators/phone_configuration_decorator.rb

```diff
@@ -1,6 +0,0 @@
-PhoneConfigurationDecorator = Struct.new(:phone_configuration) do
-  def default_number_message
-    I18n.t('account.index.default') if
-      phone_configuration == phone_configuration.user.default_phone_configuration
```

Comment:
Bullet was failing due to this being an N+1 query from loading all of the phone configurations and calling `.user` on each one. That led me down a rabbit hole of simplifying the view to do the logic directly:

```erb
          <% if current_user.default_phone_configuration == phone_configuration %>
            <%= I18n.t('account.index.default') %>
          <% end %>
```

When looking into other uses of the struct, I noticed it was being used when rendering PIV/CACs and authentication apps in a similar manner to check if it's the default. An authentication app or PIV configuration will never equal the user's default phone configuration though, so I removed those uses, and the class was no longer being used.